app/models/proposal.rb

```diff
@@ -173,6 +173,11 @@ def subscribers
     results.compact.uniq
   end
 
+  def subscribers_except_future_step_users
+    results = currently_awaiting_step_users + individual_steps.completed.map(&:user) + observers + [requester]
```

Comment:
should it be `individual_steps.completed.map(&:completed_by)` instead of `&:user` ? Do we want to send to system mailboxes or the human individuals?